lib/tabular_data/container.rb

```diff
@@ -25,6 +26,12 @@ def rows
       results
     end
 
+    # rubocop:disable all
+    def rows=(new_rows)
+      @rows = new_rows
+    end
+    # rubocop:enable all
```

Comment:
Just use [`attr_accessor :columns`](http://ruby-doc.org/core-2.0.0/Module.html#method-i-attr_accessor) at the top, and you get this method for free.